tests-performance/README.md

```diff
@@ -0,0 +1,38 @@
+### Profiling and Performance Testing of `libmata`
+
+#### Structure
+
+1. `automata/`: short database of approximately 70 automata of various complexity. The automata come from different
+  benchmarks. `easiest` benchmarks should take less then second, while, `hardest` can take over 60 seconds.
+
+2. `bin/`: target folder for compiling binaries used for profiling or performance testing.
+3. `src/`: source codes that will be compiled to `bin` directory. The directory contains several templates:
+  * `template.cc` contains basic template for arbitrary code to be profiled or performance tested.
+  * `template-with-cli.args.cc` contains basic template that takes file as an command line argument, parses the automaton
+     and computes minterms.
+  * `template-with-list-of-automata.cc` contains basic template that has a hardcoded list of input automata taken from `automata` directory.

```

Comment:
I would add a template which takes as a CLI parameter a file containing a list of paths to different automata to be tested. As with `single-automata.input`.

Reply:
Ook. Makes sense.

Reply:
It looks great. I consider this conversation to be resolved.